### DISCUSSION TITLE
Trying to collect on every run

### DISCUSSION THREAD

**Discussion by devsky047** [Taxel/PlexTraktSync#1526] [General]
My PTS has completed the initial scan, and there were numerous entries that had the warning "because provider local has no external Id" etc. It seems to try on every run to refind these. Is there a way to have it just skip over the ones it wasn't able to match and collect, etc without turning off collections in general?

**Comment by simonc56** [comment]
>  Is there a way to have it just skip over the ones it wasn't able to match

The warning literally says "_Skipping x because provider local has no external Id_" so it already does what you ask.
